README.md

```diff
@@ -1,88 +1,78 @@
 # PromoTube
 
 PromoTube is a webapp that allows users to easily find promo codes and affiliate links that go with their favorite creators or products.
+It uses the YouTube Data API to find videos from a given channel. We then parse the video descriptions and return the found promo codes.
+It also is able to find promo codes from business's in a similar way.
 
```

Comment:
Add Background section
 ## Background
YouTube content creators receive external sponsorships from companies to advertise products to their subscribers. This often comes in the form of promo codes and affiliate links which are written in the description of their YouTube video. These provide discounts for their followers while also giving some proceeds back to the creator. Currently, there exists no way for viewers or consumers to easily aggregate these discount codes without manually searching through videos. By compiling the promotional codes from a specific content creator, users will be able to more easily support their favorite YouTubers and find discounts for themselves.